addons/account_payment/static/src/js/payment_form.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import PaymentForm from '@payment/js/payment_form';
+PaymentForm.include({
+    _prepareTransactionRouteParams() {
+        let transactionRouteParams = this._super(...arguments);
+        let postParams = JSON.parse(pythonDictToJSON(this.paymentContext['postParams']));
+        transactionRouteParams = { ...transactionRouteParams, ...postParams };
```

Comment:
return directly the value here, no need for the variable right ? Also when you can add some doctstring when you create new function :smile:

Reply:
The function is just an override of another function (which has a docstring) 
Here I am just parsing the `postParams` and adding it to the result